src/World.cc

```diff
@@ -220,22 +221,26 @@ Errors World::Load(sdf::ElementPtr _sdf, const ParserConfig &_config)
     }
   }
 
+  std::unordered_set<std::string> nestedModelNames;
```

Comment:
nit: we don't usually call `//world/model`s as "nested models". Should this be `modelNames`?

Reply:
fixed in b4c352287333e85b69fa5ce96cc185a5cd9d006b